Sming/third-party/.patches/esp-open-lwip.patch

```diff
@@ -125,6 +142,24 @@ index eaa8dd6..6a08192 100644
  #endif
  #endif
  
+@@ -1823,7 +1823,7 @@
+  * debug messages of certain types.
+  */
+ #ifndef LWIP_DBG_TYPES_ON
+-#define LWIP_DBG_TYPES_ON               LWIP_DBG_OFF
++#define LWIP_DBG_TYPES_ON               LWIP_DBG_ON
```

Comment:
This is a second step to enable debugging in LWIP. It is still dependent on ENABLE_LWIPDEBUG in Sming/Makefile. In other words if ENABLE_LWIPDEBUG is 0 debugging will be disabled completely This has to remain like this to enable all debug messages. Please see macro definition of LWIP_DEBUGF in (lwip/include/lwip/debug.h)